### PR TITLE
nvme-get-property : Add documentation

### PR DESCRIPTION
**Pull request by bdeyal**
Added nvme-get-property documentation for the get-property command added at commit  460a4c0f1dbdf9aaa62932a7ec2f80a755b182f9

### CODE REVIEW

File: Documentation/nvme-get-property.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
+nvme-get-property(1)
+====================
+
+NAME
+----
+nvme-get-property - Reads and shows the defined NVMe controller property
+for NVMe ove Fabric
```
Comment (minwooim) [inline Documentation/nvme-get-property.txt]: Typo here. `ove` to `over`

File: Documentation/nvme-get-property.txt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+SYNOPSIS
+--------
+[verse]
+'nvme get-property' <device> [--offset=<offset> | -o <offset>]
+				[--human-readable | -H ]
+
+
+DESCRIPTION
+-----------
+Reads and shows the defined NVMe controller property for NVMe ove Fabric.
```
Comment (minwooim) [inline Documentation/nvme-get-property.txt]: Same typo here. `ove` to `over`

File: Documentation/cmds-main.txt [hunk 3]
```diff
@@ -124,3 +124,8 @@ linknvme:nvme-discover[1]::
 linknvme:nvme-connect[1]::
 	Connect to an NVMe-over-Fabrics subsystem
 
+linknvme:nvme-set-property[1]::
+	Writes and shows NVMe-over-Fabrics controller property
```
Comment (minwooim) [inline Documentation/cmds-main.txt]: It's very trivial, but this commit would be nicer if it holds get-property only as commit title does.

**Comment by bdeyal** [comment]
Fixed typos, removed unrelated line.

**Comment by bdeyal** [comment]
Fixed typo after comments, removed unrelated line a week ago.

**Comment by bdeyal** [comment]
Please comment if no merge, 
I will close iif so.
Thanks.

**Pull request merged**